src/wp-admin/includes/post.php

```diff
@@ -1320,11 +1325,12 @@ function wp_edit_attachments_query( $q = false ) {
 function postbox_classes( $id, $page ) {
 	if ( isset( $_GET['edit'] ) && $_GET['edit'] == $id ) {
 		$classes = array( '' );
-	} elseif ( $closed = get_user_option( 'closedpostboxes_' . $page ) ) {
+	} elseif ( get_user_option( 'closedpostboxes_' . $screen_id ) ) {
+		$closed = get_user_option( 'closedpostboxes_' . $screen_id );
 		if ( ! is_array( $closed ) ) {
 			$classes = array( '' );
 		} else {
-			$classes = in_array( $id, $closed ) ? array( 'closed' ) : array( '' );
+			$classes = in_array( $box_id, $closed, true ) ? array( 'closed' ) : array( '' );
```

Comment:
`$box_id` is not declared anywhere. This should cause some errors. 

See function in https://github.com/ClassicPress/ClassicPress/blob/97fe10d79f2875b564b5ad121dc85a726baa7b66/src/wp-admin/includes/post.php#L1325-L1351

Reply:
Good catch - that should be $id, upstream variable name change was been copied back. Updated PR to follow.